dev-proxy/ProxyCommandHandler.cs

```diff
@@ -18,6 +18,8 @@ public class ProxyCommandHandler : ICommandHandler
     public Option<IEnumerable<string>?> WatchProcessNames { get; set; }
     public Option<int?> Rate { get; set; }
     public Option<bool?> NoFirstRun { get; set; }
+    public Option<bool?> DoNotActAsSystemProxy { get; set; }
```

Comment:
Let's rename these options to be "positive". Otherwise, we end up with double negations (`!DoNotAct...`) which makes it hard to follow the code. So instead, we'd have `ActAsSystemProxy` with a default value of `true` and you could set it to `false` if you needed

Reply:
Yes, I find this clearer. Also, I wonder if there are some other, less verbose names for these options, that are just as clear. Would `--system-proxy false` and `--install-cert false` be just as clear?

Reply:
How about `--as-system-proxy` and `--install-cert`?
`--system-proxy` could be misunderstood as "do not use the currently configured system proxy".

Reply:
Good rationale. Let's use `--as-system-proxy` and `--install-cert`

Reply:
Changed with 75ff1107c3ccd13948cc18d4f9a661d3fd1f2428